vsphere/resource_vsphere_host.go

```diff
@@ -817,3 +912,206 @@ func (h HostAccessManager) ChangeLockdownMode(ctx context.Context, mode types.Ho
 	_, err := methods.ChangeLockdownMode(ctx, h.Client(), &req)
 	return err
 }
+
+func resourceVSphereHostUpdateServices(d *schema.ResourceData, meta interface{}, oldVal, newVal interface{}) error {
```

Comment:
It looks like `oldVal` and `newVal` are not used anywhere

Reply:
here is where it is being used:
	mutableKeys := map[string]func(*schema.ResourceData, interface{}, interface{}, interface{}) error{
		"services": resourceVSphereHostUpdateServices,
	}
	for k, v := range mutableKeys {
		log.Printf("[DEBUG] Checking if key %s changed", k)
		if !d.HasChange(k) {
			continue
		}
		log.Printf("[DEBUG] Key %s has change, processing", k)
		old, newVal := d.GetChange(k)
		err := v(d, meta, old, newVal)
		if err != nil {
			return fmt.Errorf("error while updating %s: %s", k, err)
		}
	}